db/seeds.rb

```diff
@@ -86,6 +83,9 @@
     {name: 'MichaelS', actionkit_uri: '/rest/v1/tag/1160/'},
     {name: 'DeborahL', actionkit_uri: '/rest/v1/tag/1661/'},
     {name: 'KatieF', actionkit_uri: '/rest/v1/tag/1662/'}
+    {name: 'ChristianB', actionkit_uri: '/rest/v1/tag/1383/'}
```

Comment:
The first two lines here need commas after them, or else they'll throw a syntax error.